app/views/static_pages/help.html.erb

```diff
@@ -6,4 +6,4 @@
   To get help on this sample app, see the
   <a href="http://www.railstutorial.org/book"><em>Ruby on Rails Tutorial</em>
   book</a>.
```

Comment:
- `<% provide(:title, "Help") %>` => `<% provide :title, t(".help") %>`
- Fix all `I18n` in this file